### ISSUE TITLE
Critical bug

### ISSUE THREAD

**Issue by talentlessguy** [jonschlinkert/union-value#4]
After installing the module I get this warning:

```
set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.
```

I haven't experienced any problems yet but `set-value` needs to be updated.

**Comment by doowb** [comment]
Thanks for the issue! Looks like #3 will address this and I'll get it published to NPM.

**Comment by doowb** [comment]
Fixed with version `2.0.1`.
